app/poros/i_scream_results.rb

```diff
@@ -0,0 +1,89 @@
+class IScreamResults 
+
+  def service
+    IScreamService.new
+  end
+
+  def game_params(game_name)
+    service.get_game_exact(game_name)
+  end
+
+  def create_game_objects(game_name)
+    data = game_params(game_name)[:data]
+    game = game_object(data)
+
+    category_objects(data, game)
+    genre_objects(data, game)
+    keyword_objects(data, game)
+    platform_objects(data, game)
+    screenshot_objects(data, game)
+    similar_objects(data, game)
+    theme_objects(data, game)
+  end
+
+  def game_object(data)
+    Game.create(new_game_params(data))
+  end
+
+  def category_objects(data, game)
+    return nil if !data[:categories].is_a?(Array)
```

Comment:
many of the games don't have certain attributes and instead of an array its `null` or `0`. In those cases it will skip the object instantiation

Reply:
this looks great!